Tasks/MSBuild/MSBuild.ps1

```diff
@@ -73,18 +77,57 @@ if ($configuration)
 
 Write-Verbose "args = $args"
 
-if(!$msBuildLocation)
+# Default the msbuildLocationMethod if not specified. The input msbuildLocationMethod
+# was added to the definition after the input msbuildLocation.
+if ($msbuildLocationMethod -ne 'location' -and $msbuildLocationMethod -ne 'version')
 {
-    if(Get-Command -Name "Get-MSBuildLocation" -ErrorAction SilentlyContinue)
+    # Infer the msbuildLocationMethod based on the whether msbuildLocation is specified.
+    if ($msbuildLocation)
     {
-        if($msbuildVersion -eq "latest")
+        $msbuildLocationMethod = 'location'
+    }
+    else
+    {
+        $msbuildLocationMethod = 'version'
+    }
+
+    Write-Verbose "Defaulted msbuildLocationMethod to: $msbuildLocationMethod"
+}
+
+# Default to 'version' if the user chose 'location' but didn't specify a location.
+if ($msbuildLocationMethod -eq 'location' -and "$msbuildLocation" -eq '')
```

Comment:
This is required now that the above fix was added for the existing radio option bug.